src/v2/view-builder/views/IdentifierView.js

```diff
@@ -62,6 +62,10 @@ const Body = BaseForm.extend({
 
     // add external idps buttons
     const idpButtons = createIdpButtons(this.options.appState.get('remediations'));
+    const pivButton = createPIVButton(this.options.settings, this.options.appState);
+    if (pivButton) {
+      idpButtons.unshift(pivButton);
```

Comment:
could we add an extra comment to it?

Reply:
What's the purpose for `unshift` ?

Reply:
From PM - When piv is present, it should be the first item in the list

Reply:
If you reuse `createIdpButtons` then we don't need to do this `unshift` here.. at can be handled by the function itself.

Reply:
I still think it is not a good idea to over-inflate `createIdpButtons`. They are different remediation form names from the widget perspective

Reply:
I think this view should not care about what PIV button is.. It should be dumb enough to just load buttons.